### PR TITLE
Translated introduction-zh.

### PR DESCRIPTION
**Pull request by Patrick-LuoYu**
Signed-off-by: Patrick-LuoYu <patrickluo@yunify.com>
/assign @Sherlock113

### CODE REVIEW

File: content/zh/docs/project-user-guide/custom-application-monitoring/introduction.md [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
+### 步骤 1：暴露 Prometheus 格式的指标
 
-First of all, your application must expose Prometheus-formatted metrics. Prometheus exposition format is the de-facto format in the realm of cloud-native monitoring. Prometheus uses a [text-based exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/). Depending on your application and use case, there are two ways to expose metrics:
+首先，您的应用必须暴露 Prometheus 格式的指标。Prometheus 展示格式已经成为云原生监控领域事实上的标准格式。Prometheus 使用[基于文本的展示格式](https://prometheus.io/docs/instrumenting/exposition_formats/)。取决于您的应用和使用场景，您可以采用以下两种方式暴露指标。
 
-#### Direct exposing
+#### 直接暴露
 
-Directly exposing Prometheus metrics from applications is a common way among cloud-native applications. It requires developers to import Prometheus client libraries in their codes and expose metrics at a specific endpoint. Many applications, such as ETCD, CoreDNS, and Istio, adopt this method.
+直接暴露 Prometheus 格式的应用指标是云原生应用的常用方式。这种方式需要开发者在代码中导入 Prometheus 客户端库并在特定的端点暴露指标。许多应用，例如 ETCD、CoreDNS 和 Istio，都采用这种方式。
```
Comment (Sherlock113) [inline content/zh/docs/project-user-guide/custom-application-monitoring/introduction.md:24]: 端点
->
端点 (Endpoint)

File: content/zh/docs/project-user-guide/custom-application-monitoring/introduction.md [hunk 2]
```diff
@@ -1,54 +1,54 @@
...
+[示例 Web 应用](../examples/monitor-sample-web)演示了如何直接暴露 Prometheus 格式的应用指标。
 
-#### Indirect exposing
+#### 间接暴露
 
-If you don’t want to modify your code or you cannot do so because the application is provided by a third party, you can deploy an exporter which serves as an agent to scrape metric data and translate them into Prometheus format.
+如果您不希望修改代码，或者由于应用由第三方提供您无法修改代码，您可以部署一个导出器作为代理，用于抓取指标数据并将其转换成 Prometheus 格式。
 
-For most third-party applications, such as MySQL, the Prometheus community provides production-ready exporters. Please refer to [Exporters and Integrations](https://prometheus.io/docs/instrumenting/exporters/) for available exporters. In KubeSphere, it is recommended to [enable OpenPitrix](../../../pluggable-components/app-store/) and deploy exporters from the App Store. Exporters for MySQL, Elasticsearch, and Redis are all built-in items in the App Store.
+Prometheus 社区为大多数第三方应用，例如 MySQL，提供了产品就绪的导出器。您可以在 [Exporters and Integrations](https://prometheus.io/docs/instrumenting/exporters/) 页面查看可用的导出器。在 KubeSphere 中，建议[启用 OpenPitrix](../../../pluggable-components/app-store/) 并从应用商店部署导出器。应用商店中内置了面向 MySQL、Elasticsearch 和 Redis 的导出器。
```
Comment (Sherlock113) [inline content/zh/docs/project-user-guide/custom-application-monitoring/introduction.md:34]: 产品就绪
->
生产就绪

File: content/zh/docs/project-user-guide/custom-application-monitoring/introduction.md [hunk 3]
```diff
@@ -1,54 +1,54 @@
...
+在 KubeSphere v3.0.0，您需要将 ServiceMonitor 和应用（或导出器）打包到 Helm Chart 中以便重复使用。在未来的版本中，KubeSphere 将提供图形化界面以方便操作。
 
-Please read [Monitor Sample Web Application](../examples/monitor-sample-web) to learn how to pack ServiceMonitor with your application.
+请参阅[监控示例 Web 应用](../examples/monitor-sample-web)了解如何打包 ServiceMonitor 和应用。
 
-### Step 3: Visualize Metrics
+### 步骤 3：实现指标可视化
 
-Around two minutes, the KubeSphere monitoring engine starts to scape and store metrics. Then you can use PromQL to query metrics and design panels and dashboards.
+大约两分钟后，KubeSphere 监控引擎开始抓取和存储指标，随后您可以使用 PromQL 查询指标并设计操作面板和仪表板。
```
Comment (Sherlock113) [inline content/zh/docs/project-user-guide/custom-application-monitoring/introduction.md:52]: 设计操作面板和仪表板。
->
设计操作面板和监控面板。

File: content/zh/docs/project-user-guide/custom-application-monitoring/introduction.md [hunk 4]
```diff
@@ -1,54 +1,54 @@
...
 
-Please read [Querying](../visualization/querying) to learn how to write a PromQL expression. For dashboard features, please read [Visualization](../visualization/overview).
\ No newline at end of file
+请参阅[查询](../visualization/querying)了解如何编写 PromQL 表达式。有关仪表板功能的更多信息，请参阅[可视化](../visualization/overview)。
```
Comment (Sherlock113) [inline content/zh/docs/project-user-guide/custom-application-monitoring/introduction.md:54]: 有关仪表板功能
->
有关监控面板功能

**Review by Sherlock113 — COMMENTED** [review]
Have a look at my comments and make changes. Thanks.

**Comment by Patrick-LuoYu** [comment]
Fixed issues based on comments.
/assign @Sherlock113

**Pull request merged**